### PR TITLE
Bl 966 fe connect and clean up admin pages

### PR DESCRIPTION
**Pull request by esmodea**
## Description

We made minor layout changes to match the more current figma and keep designs consistent across pages. We also fixed routes to allow users to navigate the admin pages to the intended routes.

[Jira Ticket](https://bloomtechlabs.atlassian.net/browse/BL-966)

## Video Log
<a>https://clipchamp.com/watch/BKaz8SFFHke<a/>

## Author & Co-Author
Jazmine Kime 
Esmodea Burk

### CODE REVIEW

**Review by AlexiusThomas — COMMENTED** [review]
I can see why you asked for more clarification.

File: src/components/common/Navbars/NavBar.js [hunk 1]
```diff
@@ -70,7 +70,7 @@ function NavBar(props) {
       style={{
         backgroundColor: bgColor,
         minHeight: '98px',
-        lineHeight: '0',
+        lineHeight: '0px',
```
Comment (ryan-hamblin) [inline src/components/common/Navbars/NavBar.js:73]: something to consider moving forward -- that is outside the scope of this ticket -- is that we should open an Issue to clean up the inline styles throughout the app.

File: src/components/pages/AdminHome/AdminSidebar.js [hunk 2]
```diff
@@ -35,45 +29,23 @@ function AdminSidebar() {
...
-          key="1"
-          icon={<ThunderboltOutlined />}
-          className="dashboard-logo"
-        >
-          Coderheroes
-        </Menu.Item>
-
-        <Menu.Item key="2" icon={<HomeOutlined />}>
-          <a href="/admin">Home</a>
+      <Menu theme="light" defaultSelectedKeys={[path]} mode="inline">
```
Comment (ryan-hamblin) [inline src/components/pages/AdminHome/AdminSidebar.js:32]: right now it's light, but will this need to be dynamic? 

the reason I'm wondering is do we need to be explicit? Or is light mode the default?

File: src/styles/AdminDashboardHome/index.less [hunk 3]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (ryan-hamblin) [inline src/styles/AdminDashboardHome/index.less:1]: did we mean to add these spaces?

File: src/styles/AdminDashboardHome/index.less [hunk 4]
```diff
@@ -12,10 +14,10 @@ h2 {
   padding-top: 0.5%;
   padding-left: 1%;
   padding-bottom: 1%;
-  font-family: Roboto;
   font-size: 30px;
 }
 
+
```
Comment (ryan-hamblin) [inline src/styles/AdminDashboardHome/index.less:20]: please remove.

File: src/styles/AdminDashboardHome/index.less [hunk 5]
```diff
@@ -36,6 +38,20 @@ h2 {
   margin-left: 10px;
 }
 
+.see-all-link {
```
Comment (ryan-hamblin) [inline src/styles/AdminDashboardHome/index.less:41]: after seeing the inline styles, I'm happy that this code is written inside a stylesheet.

File: src/styles/AdminDashboardHome/index.less [hunk 6]
```diff
@@ -36,6 +38,20 @@ h2 {
...
+  background-color: #21C5B5;
+  padding: 0px .5rem 0px .5rem;
+  border-radius: .15rem;
+  flex-shrink: 0;
+  width: 4.8rem;
+  font-size: 1rem;
+}
+
+.see-all-link:hover {
+  color: black;
```
Comment (ryan-hamblin) [inline src/styles/AdminDashboardHome/index.less:52]: lets find out if we'll need these styles for any other links within the app? If we do then we can consider styling out our links globally within the `themeOverrides.js` file.

File: src/styles/AdminStyles/index.less [hunk 7]
```diff
@@ -60,6 +60,16 @@
 .approvedBtn {
   margin: 5% 33%;
 }
+// styles for admin SideNav
+aside.ant-layout-sider{
```
Comment (ryan-hamblin) [inline src/styles/AdminStyles/index.less:64]: add space betwixt definition of the selector and the `{` curly-brace.

Comment (ryan-hamblin) [reply src/styles/AdminStyles/index.less:64]: sidenote: you may be able to achieve this by running `npm run format`

**Review by ryan-hamblin — APPROVED** [review]
there are only a couple of changes that I suggest prior to opening and then merging this PR.

File: src/components/common/Navbars/NavBar.js [hunk 8]
```diff
@@ -41,7 +41,7 @@ function NavBar(props) {
 
   useEffect(() => {
     if (role_id === 5) setBgColor('#9FB222');
-    else if (role_id < 5) setBgColor('#21C5B5');
+    else if (role_id < 5 && role_id > 2) setBgColor('#21C5B5');
```
Comment (esmodea) [inline src/components/common/Navbars/NavBar.js:44]: This was done so that only the admin's pages' headers would be changed to the set bgColor.

File: src/components/common/Navbars/NavBarLinks.js [hunk 9]
```diff
@@ -6,17 +6,17 @@ export default function NavBarLinks(props) {
...
-          COMPONENTS
+          Components
         </NavLink>
         <NavLink className="navbar__navLink" to="/admin-add-course">
-          UTILITIES
+          Utilities
         </NavLink>
         <NavLink className="navbar__navLink last_navLink" to="/">
-          PAGES
+          Pages
```
Comment (esmodea) [inline src/components/common/Navbars/NavBarLinks.js:19]: Some changes to match the header in the figma.

File: src/components/pages/AdminHome/AdminHomeContent.js [hunk 10]
```diff
@@ -15,11 +16,15 @@ const AdminHomeContent = () => {
...
+          <Link className="see-all-link" to="/admin-courses">
+            See All
+          </Link>
         </div>
         <div class="container applications">
           <h2>Instructor Applications</h2>
-          <a href="/instructors">See All</a>
+          <Link className="see-all-link" to="/admin-instructors">
+            See All
+          </Link>
```
Comment (esmodea) [inline src/components/pages/AdminHome/AdminHomeContent.js:27]: Changing things to include links speeds up navigation and stores destination to history to be more readily accessible.

File: src/components/pages/AdminHome/AdminSidebar.js [hunk 11]
```diff
@@ -1,31 +1,25 @@
...
-  ThunderboltOutlined,
-  PlusOutlined,
-  BellOutlined,
-  CalculatorOutlined,
-  FormOutlined,
-  DollarOutlined,
-  FolderOpenOutlined,
+  DashboardOutlined,
+  TableOutlined,
+  ProfileOutlined,
```
Comment (esmodea) [inline src/components/pages/AdminHome/AdminSidebar.js:8]: Removed unneeded icons from imports and changed stylesheets to the semantically correct one.

File: src/components/pages/AdminHome/AdminSidebar.js [hunk 12]
```diff
@@ -35,45 +29,23 @@ function AdminSidebar() {
...
-          <a href="/admin-purchases">Purchases</a>
-        </Menu.Item>
-
-        <Menu.Item key="6">
-          <a href="/admin-instructors">Instructors</a>
-        </Menu.Item>
+          <Menu.Item key="4">
+            <Link to="/admin-courses">Programs</Link>
+          </Menu.Item>
+        </Menu.SubMenu>
```
Comment (esmodea) [inline src/components/pages/AdminHome/AdminSidebar.js:45]: Removed deprecated sidebar items.

**Review by gclancy121 — APPROVED** [review]
Fantastic commit history Es. I really like the comments you added to each change, it makes it very clear what has been changed.

**Review by JackNunnink — APPROVED** [review]

**Review by savmaxwell93 — COMMENTED** [review]
The small changes you made to clean up the less files is appreciated and definitely improves readability. The comments you've made for each file are fantastic and make it very easy to see what was changed. Great job keeping everything clear and concise.

**Review by Jwatk13 — APPROVED** [review]
Great job overall, love the clear and concise commit messages and notes. Nice work including links to help with nav in the AdminHomeContent file!

**Review by AlexiusThomas — APPROVED** [review]

**Review by AlexiusThomas — APPROVED** [review]
Good coding and organization. very readable
